frontend/src/plugins/i18n/index.js

```diff
@@ -109,7 +116,20 @@ Object.defineProperty(i18n, 'browserPreferredLocale', {
         return languageFallback
       }
     }
-    return fallbackLocale
+    if (typeof fallbackLocale === 'string') {
+      return fallbackLocale
+    } else if (Array.isArray(fallbackLocale)) {
+      for (const fallback of fallbackLocale) {
+        return fallback
+      }
```

Comment:
Same here

Reply:
done

Reply:
This is not the same use case. Here you just select the desired language based on browser locale.